advocacy_docs/partner_docs/HashicorpVault/04-ConfiguringHashicorpVault.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: 'Configuration'
+description: 'Walkthrough on configuring the integration'
+---
+
+Implementing Hashicorp Vault with EDB Postgres Advanced Server 15.2 requires the following components installed on the same system:
+
+- EDB Postgres Advanced Server version 15.2 
+- Hashicorp Vault version 1.13.2+ent or 1.12.6+ent
+- [Pykmip](https://pypi.org/project/PyKMIP/#files)
+- Python 
+
+## Prerequisites
+
+- A running EDB Postgres Advanced Server version 15.2 instance.
+- Hashicorp Vault installed and deployed per your VM environment
+
+## Configure Hashicorp Vault KMIP Secrets Engine
+
+!!! Note
+    When you are configuring Vault commands and if at any point you receive this error message “Get "https://127.0.0.1:8200/v1/sys/seal-status": http: server gave HTTP response to HTTPS client” you need to issue this in your command line export VAULT_ADDR="http://127.0.0.1:8200".
+
+1. After your Hashicorp Vault configuration is installed and deployed per the guidelines in the [Hashicorp documentation](https://developer.hashicorp.com/vault/tutorials/getting-started/getting-started-install), you will then need to enable the kmip capabilities.
+
+2. Assume root user.
+
+3. When you are the root user, type ‘vault secrets enable kmip’ in the command line.
```

Comment:
Do we need to say "in the command line" for this and the other steps?  We are in the command line so could we not just tell them to enter ....